lib/index.js

```diff
@@ -31,6 +32,11 @@ function VM (trie, blockchain) {
   this.blockchain = this.stateManager.blockchain
   this.trie = this.stateManager.trie
 
+  // Enable homestead?
+  if (opts && opts.enableHomestead) {
+    this.enableHomestead = opts.enableHomestead
```

Comment:
lets keep the options isolated;  so move `this.enableHomestead` to `this.opts.enableHomestead` that why it will be really clear which props are to be configured